nanshe_ipython.ipynb

```diff
@@ -1173,7 +1173,21 @@
         "                da_imgs_flt.dtype.itemsize >= 4):\n",
         "            da_imgs_flt = da_imgs_flt.astype(np.float32)\n",
         "\n",
-        "        da_result = normalize_data(da_imgs)\n",
+        "        da_imgs_flt_mins = da_imgs_flt.min(\n",
+        "            axis=tuple(irange(1, da_imgs_flt.ndim)),\n",
+        "            keepdims=True\n",
+        "        )\n",
+        "        da_imgs_flt_maxes = da_imgs_flt.max(\n",
+        "            axis=tuple(irange(1, da_imgs_flt.ndim)),\n",
+        "            keepdims=True\n",
+        "        )\n",
+        "\n",
+        "        da_imgs_flt_rscl = (\n",
+        "            (da_imgs_flt - da_imgs_flt_mins) /\n",
```

Comment:
The subtraction of the minimums is handy. That said, maybe it is better to subtract the minimum of the whole image stack instead of subtracting each frame's minimum. Though this is somewhat consistent with what is being done before except using the mean of each frame instead of the minimum. Should also retain good variability in each frame.